mackerel-plugin-jvm/lib/jvm.go

```diff
@@ -341,12 +341,17 @@ func generateRemote(remote, host string, port int) string {
 
 // Do the plugin
 func Do() {
+	// Prefer ${JAVA_HOME}/bin if JAVA_HOME presents
+	pathBase := "/usr/bin"
+	if javaHome := os.Getenv("JAVA_HOME"); javaHome != "" {
+		pathBase = javaHome + "/bin"
+	}
 	optHost := flag.String("host", "", "jps/jstat target hostname [deprecated]")
 	optPort := flag.Int("port", 0, "jps/jstat target port [deprecated]")
 	optRemote := flag.String("remote", "", "jps/jstat remote target. hostname[:port][/servername]")
-	optJstatPath := flag.String("jstatpath", "/usr/bin/jstat", "jstat path")
-	optJinfoPath := flag.String("jinfopath", "/usr/bin/jinfo", "jinfo path")
-	optJpsPath := flag.String("jpspath", "/usr/bin/jps", "jps path")
+	optJstatPath := flag.String("jstatpath", pathBase+"/jstat", "jstat path ")
```

Comment:
The last space seems to be added intentionally; `"jstat path "`.

Reply:
Yes it's unintended, so I removed it. Thanks!